frontend/www/js/omegaup/components/homepage/Testimonials.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <!-- id-lint off -->
+  <div
+    class="carousel slide testimonials-carousel"
+    data-ride="carousel"
+    data-interval="8000"
+    id="testimonials"
```

Comment:
nit:

```suggestion
    id="testimonials-carousel-display"
```

el `-display` por consistencia con el del homepage.